src/Command/ReindexCommand.php

```diff
@@ -113,7 +117,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     ->getConfiguration()
                     ->setSQLLogger(new EchoSQLLogger());
             }
-            $output->writeln('Finding indexable documents...');
+            $output->writeln('Finding indexable items...');
```

Comment:
Misschien ook `$io = new SymfonyStyle($input, $output)` introduceren voor dit (en andere) commands?

Reply:
het is een "nice to have", maar geen requirement vind ik

Reply:
Het kan een requirement zijn als je deze commando's in een geautomatiseerde omgeving laat draaien, bijv. een reindex opnemen in de deployment. De output van `$output` wordt soms/altijd herkend als `stderr` en de `SymfonyStyle` heeft dat niet. Correct me if I'm wrong @7ochem :grimacing: